sky/shell/platform/android/org/domokit/sky/shell/FlutterSemanticsToAndroidAccessibilityBridge.java

```diff
@@ -117,6 +115,8 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
         // the views should be ordered top-to-bottom, tie-breaking
         // left-to-right (right-to-left in rtl environments), height,
         // width, and finally by list order.
+        // (What about something like a two-column layout? How likely is it that the
+        // natural view order isn't the right relative order of the views?)
```

Comment:
Seems reasonable. Filed https://github.com/flutter/flutter/issues/1679 and https://github.com/flutter/flutter/issues/1678.